xun/tests/test_graph.py

```diff
@@ -0,0 +1,120 @@
+from xun.functions.graph import CallNode
+from xun.functions.graph import CallNodeSubscript
+
+
+def test_unpack():
+    cn = CallNode('f')
+
+    assert CallNodeSubscript(cn, (0, )) == CallNodeSubscript(cn, (0, ))
+    assert not CallNodeSubscript(cn, (0, )) == CallNodeSubscript(cn, (1, ))
+
+    shape = (1, )
```

Comment:
This should be an interable, containing one item

```suggestion
    shape = (1, ) # CallNode value is iterable
```